src/Radio.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { color } from 'styled-system'
+import Icon from './Icon'
+import Text from './Text'
+import Input from './Input'
+
+const propTypes = {
+  checked: PropTypes.bool,
+  disabled: PropTypes.bool,
+  name: PropTypes.string,
+  value: PropTypes.string
+}
+
+const Radio = props => {
+  const { className, checked, disabled, name, value } = props
+
+  const radioIconName = checked ? 'radioFilled' : 'radio'
+
+  const testID = checked ? 'SELECTED' : disabled ? 'DISABLED' : 'NOT_SELECTED'
+
+  return (
+    <RadioWrap className={className} checked={checked} disabled={disabled}>
```

Comment:
I think we want to avoid exposing `className` outside of this component since we're using styled-components, right?